components/tools/ToolsCard.js

```diff
@@ -71,37 +74,48 @@ export default function ToolsCard({ toolData }) {
                 />
               ))}
             </div>
-          </div></>}
+          </div>}
+        </div> :
+        <div className="w-full relative p-8 text-center h-full text-gray-700">
+          <div className='absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2'> No Content to Display </div>
```

Comment:
I don't think we should display it.

of both language and technology is not provided, just leave blank

Reply:
If we don't display anything, many visitors may consider it a bug in displaying the tool information. It's better we tell them in card that no further information is available with us. That's why I made this case.

Reply:
if this is your reason then rename from ` No Content to Display ` to ` No further details provided ` or something like that